### PR TITLE
Updates.

### PR DESCRIPTION
**Pull request by alunduil**
These updates are long overdue.

### CODE REVIEW

File: zfs/replicate/compress/__init__.py [hunk 1]
```diff
@@ -1,4 +1,4 @@
 """ZFS Replication Stream Compression."""
 
-from .command import command
-from .type import Compression
+from .command import command as command  # pylint: disable=useless-import-alias
```
Comment (Flameeyes) [inline zfs/replicate/compress/__init__.py:3]: Maybe disable useless-import-alias on the `setup.cfg` instead?

Comment (alunduil) [reply zfs/replicate/compress/__init__.py:3]: I usually like leaving things explicit but I'll consider that.  What's the state of the art on linting Python code?  Should I just drop pylint in favour of something else instead?

Comment (Flameeyes) [reply zfs/replicate/compress/__init__.py:3]: I've been mostly relying on flake8 only, configured with [pre-commit](https://pre-commit.com) together with isort, black and reuse.

Comment (alunduil) [reply zfs/replicate/compress/__init__.py:3]: I'll setup a new pull request to change to flake8 with pre-commit.  Thanks!

**Pull request merged**